install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Based on Deno installer: Copyright 2019 the Deno authors. All rights reserved. MIT license.
+
+set -e
+
+case $(uname -sm) in
+"Darwin x86_64") target="Darwin_x86_64" ;;
+"Darwin arm64") target="Darwin_arm64" ;;
+"Linux aarch64") target="Linux_arm64" ;;
+*) target="Linux_i386" ;;
+esac
+
+version=${1}
```

Comment:
Optionally a git release tag can be provided to the script. Otherwise the latest release is used.